src/components/HotelsPagination/HotelsPagination.jsx

```diff
@@ -1,15 +1,42 @@
+/* eslint-disable no-console */
 import './StylesPagination.css';
+import PropTypes from 'prop-types';
+import HotelCard from '../HotelCard/HotelCard';
 
-const HotelsPagination = () => (
+const HotelsPagination = ({
+  results, currentPage, handleNext, handlePrev,
+}) => (
   <section className="pagination__card">
-    <div className="pagination__buttons">
-      <button className="pagination__button" type="submit">{'<<'}</button>
-      <button className="pagination__button" type="submit">1</button>
-      <button className="pagination__button" type="submit">2</button>
-      <button className="pagination__button" type="submit">3</button>
-      <button className="pagination__button" type="submit">{'>>'}</button>
-    </div>
+    {console.log('Current Page', currentPage)}
+    <section className="page__hotelsList">
+      {results.map((hotel) => (
+        <HotelCard
+          imageProfile={hotel.imageProfile}
+          name={hotel.name}
+          about={hotel.about}
+          city={hotel.city}
+          pricePerNight={hotel.pricePerNight}
+          offerPrice={hotel.offerPrice}
+          feature1={hotel.feature1}
+          feature2={hotel.feature2}
+          id={hotel._id}
+          key={hotel._id}
+        />
+      ))}
+    </section>
+    <section className="pagination__buttons">
+      <button className="pagination__button" type="submit" onClick={handlePrev}>{'<<'}</button>
+      <p className="pagination__currentPage">{currentPage + 1} of 10</p>
```

Comment:
'10' should not be harcoded. It should display the amount of pages available.

Reply:
Done!